pkgs/top-level/all-packages.nix

```diff
@@ -17793,7 +17793,9 @@ with pkgs;
   coq_8_4 = coqPackages_8_4.coq;
   coq_8_5 = coqPackages_8_5.coq;
   coq_8_6 = coqPackages_8_6.coq;
-  coq_HEAD = callPackage ../applications/science/logic/coq/HEAD.nix {};
+  coq_HEAD = callPackage ../applications/science/logic/coq {
+    version = "8.7pre-2017-05-28";
```

Comment:
https://github.com/NixOS/nixpkgs/issues/26108 and https://github.com/NixOS/nixpkgs/pull/26173